src/app/controllers/surelia.js

```diff
@@ -755,6 +755,9 @@ Surelia.prototype.retrieveMessage = function(id, boxName){
 }
 
 Surelia.prototype.getAttachment = function(attachment) {
+  if (!attachment.ready) {
+    return;
```

Comment:
@mdamt No, since no other function need those value. It only terminates the function.